cmd/cert-checker/main.go

```diff
@@ -140,23 +139,23 @@ func newChecker(saDbMap certDB,
 	}
 }
 
-func (c *certChecker) getCerts(ctx context.Context, unexpiredOnly bool) error {
-	c.issuedReport.end = c.clock.Now()
-	c.issuedReport.begin = c.issuedReport.end.Add(-c.checkPeriod)
-
-	args := map[string]interface{}{"issued": c.issuedReport.begin, "now": 0}
-	if unexpiredOnly {
-		args["now"] = c.clock.Now()
-	}
+func (c *certChecker) getCerts(ctx context.Context) error {
+	c.issuedReport.end = c.clock.Now().Truncate(time.Second).Add(time.Second)
+	c.issuedReport.begin = c.issuedReport.end.Add(-c.checkPeriod).Truncate(time.Second)
 
 	var sni sql.NullInt64
 	var err error
 	var retries int
 	for {
 		sni, err = c.dbMap.SelectNullInt(
 			ctx,
-			"SELECT MIN(id) FROM certificates WHERE issued >= :issued AND expires >= :now",
-			args,
+			`SELECT MIN(id) FROM certificates
+				WHERE issued >= :begin AND
+					  issued < :end`,
+			map[string]interface{}{
+				"begin": c.issuedReport.begin,
+				"end":   c.issuedReport.end,
```

Comment:
I recommend not using `issuedReport.end` here and instead use something much narrower than 72h while we're at it. 
Maybe `c.issuedReport.begin.Add(time.Hour)` ?

Reply:
Good idea. Done!